.vscode/settings.json

```
@@ -8,7 +8,7 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   // python
-  "python.formatting.provider": "black",
+  "python.formatting.provider": "none",
```

```suggestion
  "python.formatting.provider": "black",
```
こちらの値がnoneに書き換わっています。
「関係ないところは変えない」スタンスでお願いします。